test/spec/modules/adglareBidAdapter_spec.js

```diff
@@ -0,0 +1,98 @@
+import {expect} from 'chai';
+import {spec} from 'modules/adglareBidAdapter';
+
+describe('AdGlare Adapter Tests', function () {
```

Comment:
Add tests for `buildRequests` function

Reply:
A test for buildRequests() has been added.